tools/rpki-rov.c

```diff
@@ -141,17 +152,46 @@ int main(int argc, char *argv[])
 		char *input_tok = NULL;
 
 		input_tok = strtok(input, delims);
-		strcpy(ip, input_tok);
+		struct lrtr_ip_addr pref;
+		char ip[INET6_ADDRSTRLEN];
+
+		if (strlen(input_tok) > sizeof(ip) - 1) {
+			printf("Error: Invalid ip addr\n");
```

Comment:
maybe use `fprintf(stderr, ...)`  here and below?

Reply:
good idea, will also remove the flush calls, as they are not necessary with stderr.